### ISSUE TITLE
Key 'utility_repeater' is not properly formatted.

### ISSUE THREAD

**Issue by TiBarification** [Kwoth/NadekoBot#1235]
```
I can't tell you if the command is executed, because there was an error printing out the response. Key 'utility_repeater' is not properly formatted. Please report this.
```
Language: Russian

**Comment by Kwoth** [comment]
Thank you, I just updated the string, it will be fixed in 1.41 (next release)

**Comment by TiBarification** [comment]
@Kwoth in addition to this, here next phrase with error **gambling_slot_two**.
```
I can't tell you if the command is executed, because there was an error printing out the response. Key 'gambling_slot_two' is not properly formatted. Please report this.
```

**Comment by Kwoth** [comment]
fixed both in 1.4
